dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/support/MergeableClusterInvoker.java

```diff
@@ -182,6 +181,15 @@ protected Result doInvoke(Invocation invocation, List<Invoker<T>> invokers, Load
         return AsyncRpcResult.newDefaultAsyncResult(result, invocation);
     }
 
+    private void checkNoProvider(RpcException e, String invokerGroup) throws RpcException {
+        if (e.isNoInvokerAvailableAfterFilter()) {
+            log.debug("No available provider for service" + getUrl().getServiceKey() + " on group " + invokerGroup
```

Comment:
Should this log being updated?

Reply:
@AlbumenJ @namelessssssssssss 
If there is no merger, it makes sense to debug because it will log every call, and if there is a merger, you can consider adjusting the level to warn/info.